integration-cli/docker_cli_rename_test.go

```diff
@@ -97,7 +97,7 @@ func TestRenameCheckNames(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	logDone("rename - running container")
+	logDone("rename - check the first container's status")
```

Comment:
Hi cpuguy83,
TestRenameRunningContainer and TestRenameCheckNames  are two functions. They are included in docker_cli_rename_test.go.

func TestRenameRunningContainer(t *testing.T) {
	...
	logDone("rename - running container")
}

func TestRenameCheckNames(t *testing.T) {
	...
	logDone("rename - check the first container's status")
}

However, The two functions output the same log. So I try changing log output of one function.